README.md

```diff
@@ -1,5 +1,10 @@
 # hugegraph-client
 
+[![License](https://img.shields.io/badge/license-Apache%202-0E78BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+[![Build Status](https://travis-ci.org/hugegraph/hugegraph-client.svg?branch=master)](https://travis-ci.org/hugegraph/hugegraph-client)
+[![codecov](https://codecov.io/gh/hugegraph/hugegraph-client/branch/master/graph/badge.svg)](https://codecov.io/gh/hugegraph/hugegraph-client)
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.baidu.hugegraph/hugegraph-client/badge.svg)](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-client/1.5.7)
```

Comment:
remove 1.5.7 on branch master.
but should add specified version for release branch:
https://maven-badges.herokuapp.com/maven-central/com.baidu.hugegraph/hugegraph-client/badge.svg?version=1.5.6
ref: https://github.com/jirutka/maven-badges/pull/19